helm/cloud-provider-cloud-director-app/values.yaml

```diff
@@ -0,0 +1,27 @@
+basicAuthSecret:
+  create: true
+  username: ""
+  password: ""
+  refreshToken: ""
+
+cloud-director-named-disk-csi-driver:
+  basicAuthSecret:
```

Comment:
is not repetitive? take into account from child chart you can access a property of the parent chart using the `global` var

Reply:
This is to give the option to do either:

- Create the secret in the child chart to install only the CPI or only the CSI for instance
- Create the secret in the parent chart to be used by both the CSI and CPI and not in the child charts.

This makes the child charts as generic as possible if we want to submit them upstream for instance while having the possibility of doing it globally with the parent chart.